test_runner/fixtures/utils.py

```diff
@@ -56,6 +56,11 @@
 )
 # fmt: on
 
+no_default_lfc_env = os.environ.get("NO_DEFAULT_LFC")
+NO_DEFAULT_LFC = (no_default_lfc_env is not None) and (
+    no_default_lfc_env.lower() in {"yes", "true"}
```

Comment:
For the most of other env vars we use `true`/`false`, let's keep it consistent

Reply:
This code was removed due to refactoring.